<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="XE-GGN-IT-01460" failures="2" tests="2" name="RetryMultipleFailedTestCases" time="0.214" errors="0" timestamp="2024-12-23T23:17:23 IST">
  <testcase classname="MyTest.Test12" name="Test1" time="0.057">
    <failure type="java.lang.AssertionError" message="expected:&amp;lt;false&amp;gt; but was:&amp;lt;true&amp;gt;">
      <![CDATA[java.lang.AssertionError: expected:<false> but was:<true>
at MyTest.Test12.Test1(Test12.java:12)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- Test1 -->
  <testcase classname="MyTest.Test12" name="Test2" time="0.01">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at MyTest.Test12.Test2(Test12.java:19)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- Test2 -->
</testsuite> <!-- RetryMultipleFailedTestCases -->
